modules/microshift-version-api.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * microshift_support/microshift-version.adoc
+
+[id="microshift-version-api"]
+= Checking the {product-title} Version via the API
+
+.Procedure
+
+* Using the API, access the `kube-public/microshift-version` ConfigMap to retrieve the same information.
+
+[source,terminal]
+----
+$ oc get configmap -n kube-public microshift-version -o yaml
+----
+
```

Comment:
@jldohmann in
https://github.com/openshift/openshift-docs/pull/50054#discussion_r1010985701 you say that we need a `+` between these parts of the procedure, and I can't quite work out how to add it properly.

If I make line 16 contain only `+` then it breaks the way the "Example output" is rendered.

If I add a line with `+` and a blank line before and after, I see the `+` in output.

Can you help me with the right syntax, please?

Reply:
A `+` on both line 11 and line 16 should work.

Reply:
I can fix those in a follow-up PR.